src/modules/Builder/features/ActivityAbout/ActivityAbout.tsx

```diff
@@ -95,6 +98,15 @@ export const ActivityAbout = () => {
     },
   ];
 
+  const handleIsReviewableChange = (event: ChangeEvent<HTMLInputElement>) => {
+    if (!event.target.checked) return;
+
+    const activityKey = getEntityKey(activity || {});
```

Comment:
maybe it worth to add early return before it:
```
if (!activity) return; // or `setValue('activityFlows', activityFlows);` - if needed
```
```suggestion
    const activityKey = getEntityKey(activity);
```